src/helpers.py

```diff
@@ -5,11 +5,18 @@
 import os
 import subprocess
 from datetime import datetime
+import logging
 
 import requests
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
+log_level = os.getenv("LOG_LEVEL", logging.INFO)
+
+# Initialize Logger
+logger = logging.getLogger(__name__)
+logger.setLevel(log_level)
```

Comment:
Doesn't the setLevel method require a numeric value? 
Something like:
numeric_level = getattr(logging, log_level.upper(), None)
# ignore invalid log level setting
    if isinstance(numeric_level, int):
        logger.setLevel(numeric_level)

https://docs.python.org/3/library/logging.html#levels

Reply:
@schans Looking at the code for `setLevel` it looks `int` and `str`are supported: 

```python
def setLevel(self, level):
        """
        Set the logging level of this logger.  level must be an int or a str.
        """
        self.level = _checkLevel(level)
        self.manager._clear_cache()

def _checkLevel(level):
    if isinstance(level, int):
        rv = level
    elif str(level) == level:
        if level not in _nameToLevel:
            raise ValueError("Unknown level: %r" % level)
        rv = _nameToLevel[level]
    else:
        raise TypeError("Level not an integer or a valid string: %r" % level)
    return rv
`
``

Reply:
Today I learned ;-) Even better. We could consider some error handling though.

Reply:
May I suggest something like:

```python
# Initialize Logger
logger = logging.getLogger(__name__)

# Set log level
try:
    log_level = os.getenv("LOG_LEVEL", logging.INFO)
    log_level = log_level.upper() if isinstance(log_level, str) else log_level
    logger.setLevel(log_level)
except (ValueError, TypeError) as e:
   logger.error(f"Unable to set log level: {e}")
```

Reply:
Thanks for the suggestion @schans 👍